code/datums/diseases/transformation.dm

```diff
@@ -124,6 +124,7 @@
 	if(ishuman(affected_mob))
 		var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSTUNS | TR_KEEPREAGENTS | TR_KEEPSE)
 		M.ventcrawler = VENTCRAWLER_ALWAYS
+		M?.mind.language_holder = new M.initial_language_holder(M.mind) // make sure they speak normal monkey

```

Comment:
The question mark is at the wrong place.

Reply:
oh oops

Reply:
The issue with doing this is it overwrites the mind language holder, while the monkey language is supposed to be given to you via the mob language holder. This causes issues with removing the monkey language if this mob is ever humanized for example through genetics, as well as removes all the languages you knew as a human. Curators would lose their omnitongue for example. 
Its a bandaid, and we are just digging in at the bottom of the rabbit hole. Whenever the underlying problem with language holders and transforms is fixed, this will need to be removed again.